src/app/index.html

```diff
@@ -1,9 +1,27 @@
-<app-nav>
-</app-nav>
+<sky-navbar>
+  <div class="sky-container">
+    <sky-navbar-item>
+      <a class="active" href="/">Home</a>
+    </sky-navbar-item>
+  </div>
+</sky-navbar>
```

Comment:
Nope; I'd like 100% code coverage until we figure out what we're doing about "demo" files.